meetings-meetups/_posts/2020-03-26-virtual-foss-hours.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: redirect
+redirect: https://fossrit.github.io/
+date-start: "2020-03-26 17:00"
+date-end: "2020-03-26 18:00"
+location: "Online"
```

Comment:
Could you please change this to the Jitsi room URL?

Reply:
@ct-martin The calendar implementations I am familiar with expose the `location` metadata more visibly than the description. I would like it to be the Jitsi room URL. I know for me, I place URLs there in calendar invites because it makes it easier for invitees to find the link when I am suddenly trying to find the link to join the meeting.

Reply:
Done, but for reference sake, here's the message I put on the commit:

> Note: while it seems that it might _technical_ work, I can not comfortably say that putting a normal URL in the `location` field is a good idea nor best practice. As far as I can tell, this behavior seems to work on at least some calendar clients, but is not definitively to-spec.
>
> Based on what I can find online, the best practice would be to put something to the effect of "Jitsi/Online" in the `location` field and the URL in the `url` field (populated by the `redirect` front matter) as well as in the description/summary (since not all clients support the `url` field, including Google).

Reply:
@ct-martin Thanks for documenting this in the commit message. :+1: